Tests/DatadogIntegrationTests/Scenarios/RUM/RUMResourcesScenarioTests.swift

```diff
@@ -61,9 +61,9 @@ class RUMResourcesScenarioTests: IntegrationTests, RUMCommonAsserts {
         let firstPartyBadResourceURL = URL(string: "https://foo.bar")!
 
         // Requesting this third party by the app should create the RUM Resource.
-        let thirdPartyGETResourceURL = URL(string: "https://httpbin.org/get")!
```

Comment:
/comment
Interesting, in past this has been quite reliable from experience.

Reply:
Maybe it's too popular now!
As @fuzzybinary mentioned, it would be nice to not hit a remote endpoint, we could use the httpbin docker image to start the service locally!